Formula/n/ntopng.rb

```diff
@@ -2,11 +2,10 @@ class Ntopng < Formula
   desc "Next generation version of the original ntop"
   homepage "https://www.ntop.org/products/traffic-analysis/ntop/"
   license "GPL-3.0-only"
-  revision 2
 
   stable do
```

Comment:
Does it still need the patch? If it does, can this be upstreamed so we know we can remove it in the future?